boards/px4/fmu-v5/src/spi.cpp

```diff
@@ -266,16 +324,32 @@ __EXPORT uint8_t stm32_spi6status(FAR struct spi_dev_s *dev, uint32_t devid)
  *
  ************************************************************************************/
 
+#define _PIN_OFF(def) (((def) & (GPIO_PORT_MASK | GPIO_PIN_MASK)) | (GPIO_INPUT|GPIO_PULLDOWN|GPIO_SPEED_2MHz))
```

Comment:
These needs to be tested with a scope! Also have a look at v5x it had multiple power domains.